mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
```

Comment:
In `Profile.js`, you are displaying the user's name twice, and there's an unused `address` property. It seems like you might be trying to display additional user information.

To correct this, replace:

```javascript
<h1> Name :{this.props.userData.name} </h1>
<h1> Name :{this.props.userData.address} </h1>
```

with the actual property from `this.props.userData` that you want to display. For example, if you want to show the user's location, you'd use:

```javascript
<h1> Name :{this.props.userData.name} </h1>
<h1> Location: {this.props.userData.location} </h1>
```

Ensure that the property you're trying to access (`address` in this case) actually exists in `this.props.userData` and is being fetched correctly.